cms/static/cms/js/build.js

```diff
@@ -81,6 +81,7 @@
          */
         paths: {
             'gettext': 'empty:',
+            'json2': 'empty:',
```

Comment:
Reviewers: this change isn't causing the bug, but I discovered it while analyzing the RequireJS Optimizer output. This declares that the file json2.js does not need to be included in any bundles because it will already be included on the page.